### PR TITLE
Attendance component scaffolded out

### PR DESCRIPTION
**Pull request by justinpeisker**
## Description

This PR adds an attendance page to the sidebar.  Further functionality fleshed out in the near future(next week).

#### Video Link
https://www.loom.com/share/c3dc23d7568d4dd4b19b8f28e4660983

[Loom Video](Add your loom video link here)

#### Trello Link

https://trello.com/c/u0rWH6ac

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by jonathan-mcgraw — APPROVED** [review]
I love it. Thank you for including a Loom video. It really helps the DS students understand what's going on.

**Review by maidiance — APPROVED** [review]
Looks like a great scaffold, good work!

- Imports are setup in appropriate components
- Made a PrivateRoute that allows proper roles access to the component
- Added to the sidebar as expected for the feature

File: src/components/pages/Attendance/attendance.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Attendance = () => {
+  return (
+    <div>
```
Comment (ryan-hamblin) [inline src/components/pages/Attendance/attendance.js:5]: I love here that you're using good practices in terms of stubbing out work that will continue to be worked on. 

The only thing I would like to change is the addition of a `div` that seemingly isn't needed. I would a fragment instead `<>`.

File: src/index.js [hunk 2]
```diff
@@ -120,6 +121,12 @@ function App() {
           component={() => <div>"My Mentees" Component goes here</div>}
         />
 
+        <PrivateRoute
+          path="/resources"
+          redirect="/dashboard"
+          allowRoles={[1, 2, 3, 4]}
+          component={ManageResources}
+        />
```
Comment (ryan-hamblin) [inline src/index.js:129]: Please add a line break here between these lines so that it matches the rest of the file's composition.

**Review by ryan-hamblin — CHANGES_REQUESTED** [review]
Overall this Pull-request is clean. There are a few that I have pointed out directly inline that I would love addressed. 

- Please go and fix the linter warnings listed in by our GitHub action. They're easy to find if you open the `Files Changed` tab and scroll down bit you'll see a Warning from a check that looks like this: 

image


The way to fix those warnings is go to the file and do what the linting warning says. 

For the one above, you'd simply delete the definition of `PendingApplications` in the file. 

Remember, if we're not using it, we don't want it in `main`.

File: src/components/pages/Attendance/attendance.js (hunk 1 above)
Comment (justinpeisker) [reply src/components/pages/Attendance/attendance.js]: Thanks for the feedback Ryan.  I just updated the attendance.js to reflect your change recommendations.

File: src/index.js (hunk 2 above)
Comment (justinpeisker) [reply src/index.js:129]: Added line break to match formatting

**Comment by justinpeisker** [comment]
> Overall this Pull-request is clean. There are a few that I have pointed out directly inline that I would love addressed.
> 
> * Please go and fix the linter warnings listed in by our GitHub action. They're easy to find if you open the `Files Changed` tab and scroll down bit you'll see a Warning from a check that looks like this:
> 
> image
> 
> The way to fix those warnings is go to the file and do what the linting warning says.
> 
> For the one above, you'd simply delete the definition of `PendingApplications` in the file.
> 
> Remember, if we're not using it, we don't want it in `main`.

**Comment by justinpeisker** [comment]
Ok Ryan.  I believe I've made all the changes you've requested.  Please let me know if there is anything else to change for this PR to be merged.  Thanks!

**Review by ryan-hamblin — APPROVED** [review]
Awesome! Thanks for cleaning those things up! Keep cleaning up the linting warnings.

**Review by adamhinton — APPROVED** [review]
Looks good to me and I appreciate the good commit descriptions.

File: src/index.js [hunk 3]
```diff
@@ -30,6 +29,7 @@ import Calendar from './components/common/Calendar';
 import MentorMenteeMatching from './components/pages/MentorMenteeMatching/MentorMenteeMatching';
 import Notes from './components/pages/Notes/Notes';
 import NotesForm from './components/pages/Notes/NotesForm';
+import Attendance from './components/pages/Attendance/attendance';
```
Comment (phillipgalbraith) [inline src/index.js:32]: Why is the attendance component file lowercase? attendance.js should probably be Attendance.js

**Review by phillipgalbraith — COMMENTED** [review]
It is looking clean.  My only concern was about capitalization of the attendance.js.

**Pull request merged**